src/test/java/org/openmrs/contrib/qaframework/page/AllergiesPage.java

```diff
@@ -10,6 +10,15 @@
 			.xpath("//*[contains(text(),'No Known Allergy')]");
 	private static final By DELETE_NO_KNOWN = By.className("delete-action");
 
+	public static final By CHOOSEDRUG = By.id("allergen-162299");
```

Comment:
these elements are data dependent, use references that aren't tied onto ids since they never exist on another instance